kd-mail-server
2018-11-22 20:54:55,681 INFO 126124[no_cat_release.py:25 checkfile]: right way /mnt/data1/release-data/kd-mail-server_release/v3.1.0.RELEASE/kd-mail-server.jar
2018-11-22 20:54:55,681 INFO 126124[no_cat_release.py:31 stop_cmd]: now rm bak on the host : 10.11.5.80
2018-11-22 20:54:55,693 INFO 126124[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-22 20:54:55,777 INFO 126124[transport.py:1746 _log]: Authentication (publickey) successful!
2018-11-22 20:54:55,941 INFO 126124[no_cat_release.py:37 stop_cmd]: error list rm bak is :[]
2018-11-22 20:54:55,942 INFO 126124[no_cat_release.py:38 stop_cmd]: stdout is rm bak is :[]
2018-11-22 20:54:55,942 INFO 126124[no_cat_release.py:40 stop_cmd]: now stop the host : 10.11.5.80
2018-11-22 20:54:56,124 INFO 126124[no_cat_release.py:44 stop_cmd]: cmd is [sudo /data/CI/kd-mail-server_80_13980/mail.sh stop]
2018-11-22 20:54:56,124 INFO 126124[no_cat_release.py:45 stop_cmd]: error list stop is :[]
2018-11-22 20:54:56,125 INFO 126124[no_cat_release.py:46 stop_cmd]: stdout list stop is :[u'/data/CI/kd-mail-server_80_13980\n', u'kd-mail-server.jar\n', u'/data/CI/kd-mail-server_80_13980/lib/kd-mail-server.jar (13980) shutdown.\n']
2018-11-22 20:55:06,192 INFO 126124[no_cat_release.py:52 stop_cmd]: cmd is [sudo mv /data/CI/kd-mail-server_80_13980/lib/kd-mail-server.jar /data/CI/kd-mail-server_80_13980/lib/kd-mail-server.jar_bak]
2018-11-22 20:55:06,193 INFO 126124[no_cat_release.py:53 stop_cmd]: error list mv bak is :[]
2018-11-22 20:55:06,193 INFO 126124[no_cat_release.py:54 stop_cmd]: stdout list mv bak is :[]
2018-11-22 20:55:06,193 INFO 126124[no_cat_release.py:58 release_service]: now distribution to the host :10.11.5.80
2018-11-22 20:55:06,202 INFO 126124[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-22 20:55:06,285 INFO 126124[transport.py:1746 _log]: Authentication (publickey) successful!
Traceback (most recent call last):
  File "/home/op/mesh_deploy_5.80/no_cat_release.py", line 91, in <module>
    release_service(ip , krs_sh, krs_jar, krs_jar_bak,release_path, lib_path, server)
  File "/home/op/mesh_deploy_5.80/no_cat_release.py", line 61, in release_service
    scp.send_file(jar_file, preserve_times=True)
  File "/usr/local/lib/python2.7/dist-packages/scpclient.py", line 301, in _send_file
    with open(local_filename, 'rb') as fl:
IOError: [Errno 13] Permission denied: '/mnt/data1/release-data/kd-mail-server_release/v3.1.0.RELEASE/kd-mail-server.jar'
[Finish] Failed to deploy [419]